CodeYourFuture/cyf-forms
# Feature/697-move-employer-names-to-backend-code[QA]

This is a:





### Description

**Purpose** 

The purpose of this PR is that the employers' names list should come from the back-end code. A new volunteer should be able to see all the employers' names on the volunteer's form and be able to select if their employer's name is on the list.

**How to check** 

- Visit this branch: feature/697-move-employer-names-to-backend-code-reduced-scope.
- Run cyf-api repo.
- Run your cyf-forms repo, when the form loads you should be able to see all the employers' names sorted alphabetically.


### Links

1. issue: CodeYourFuture/tech-team#697.


2. Implementing current required changes

- cyf-api: https://github.com/CodeYourFuture/cyf-api/pull/637

3. FYI: Reverting all changes merged to the QA branch

- cyf-form:https://github.com/CodeYourFuture/cyf-forms/pull/153
- cyf-api: https://github.com/CodeYourFuture/cyf-api/pull/635


### Author checklist


- [x] I have made the PR to `qa` from a branch named **feature/697-move-employer-names-to-backend-code-reduced-scope**
- [x] I have completed the manual tests [described here](https://github.com/CodeYourFuture/tech-team/wiki/3.-How-we-work#manual-test-procedures)
